styleguide/derek/_partials/solutions/testimonials.ejs

```diff
@@ -1,26 +1,47 @@
+<%
+// While seemingly redundant, this is actually necessary for EJS to evaluate variable scope.
+var isSubpanel = !!isSubpanel;
```

Comment:
Considering this snippet:

```ejs
<%- partial(file_left, {isSubpanel: true}) %>
```

Does the value of `isSubpanel` come in as the string `'true'` or the boolean `true`?